cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,172 @@
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("#edit-submit-cgov-image-media-browser").click();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').click({ force: true }).find("input[name*='entity_browser_select'][class='form-checkbox']").check();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id='edit-submit'][value='Select image']").click({ force: true })
+})
+
+And('user selects {string} from the list of media', (image) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input#edit-name").type(image);
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("#edit-submit-cgov-media-browser").click();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').click().find("input[name*='entity_browser_select'][class='form-checkbox']").check();
+    });
+
+And('user clicks on {string} button to select media', (media) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find(`input#edit-submit[value="${media}"]`).click({ force: true })
+})
+
+And('user selects {string} from Contents dropdown', (selectOption) => {
+    cy.get(`.placeholder:contains("Contents")`).parent().find(`input[value="${selectOption}"]`).click({ force: true });
+    
+})
+
+And('user selects {string} from {string} dropdown {string} section', (dropDown, option, section) => {
+    cy.get(`div:contains("${section}")`).parent().parent().find(`.placeholder:contains("${option}")`).parent().find(`input[value="${dropDown}"]`).eq(0).click({ force: true })
+})
+
+Then('lead image is displayed with source {string}', (image) => {
+    cy.get('.centered-element img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+And('caption reads {string}', (caption) => {
+    cy.get('.caption-container p').should('include.text',caption);
+});
+
+And('credit reads {string}', (credit) => {
+    cy.get('.image-photo-credit').should('include.text',credit);
+});
+
+And('{string} button is displayed', (enlarge) => {
+    cy.get('.article-image-enlarge').should('be.visible').should('have.text', enlarge);
+});
+
+Then('feature card image is displaying {string}', (image) => {
+    cy.get('.feature-primary.flex-columns.row img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+Then('{int} item in the list has an image {string}', (item,image) => {
+    cy.get('.image.item-image > img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+Then('the {string} appear as override', (override) => {
+     cy.get('div.multimedia-slot img').first().should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(override.replace('.svg', ''));
```

Comment:
your override parameter doesn actually have .svg extension - so I dont think you need to do a replace in both expect and condition - its redundant

Reply:
fixed